src/twisted/protocols/test/test_tls.py

```diff
@@ -72,11 +74,7 @@ class HandshakeCallbackContextFactory:
         is done.
     """
 
-    # pyOpenSSL needs to expose this.
-    # https://bugs.launchpad.net/pyopenssl/+bug/372832
-    SSL_CB_HANDSHAKE_DONE = 0x20
-
-    def __init__(self, method=TLSv1_METHOD):
+    def __init__(self, method=TLS_METHOD):
```

Comment:
Minimum pyOpenSSL of [21.0.0](https://github.com/pyca/pyopenssl/releases/tag/21.0.0)